regolith/app.py

```diff
@@ -59,6 +61,17 @@ def collection_page(dbname, collname):
                     f.write(form['body'])
                 traceback.print_exc()
                 raise
+            tv, errors = validate(dbname, body)
+            if not tv:
+                td = tempfile.TemporaryDirectory()
+                n = os.path.join(td.name, 'regolith.txt')
+                print(errors)
+                print('Writing text file to {}. '
+                      'Please try again.'.format(n))
+                with open(n) as f:
```

Comment:
This file isn't opened in write mode. Do you have any tests for the invalid case?

Reply:
I don't think we have any tests for the app.

Reply:
Fair point!

Reply:
Made an issue https://github.com/regro/regolith/issues/69